File: A1/A1-018/A1-018.py
# --------------------------------------------------
# File Name : A1-018.py
# Problem   : Roman Numbers
# Author    : Worralop Srichainont
# Date      : 2026-02-08
# --------------------------------------------------

# List of Roman numbers
ROMAN_NUMS = ["I", "II", "III", "IV", "V", "VI", "VII", "VIII", "IX"]

# Input an Arabic number
num = int(input())

# Output a Roman number
if 1 <= num <= 9:
    print(ROMAN_NUMS[num - 1])
elif num < 0:
    print("Error : Please input positive number")
else:
    print("Error : Out of range")

File: A1/A1-018/A1-018.cpp
#include<iostream>
using namespace std;

int main()
{
    // This makes std::cin and std::cout faster
    ios_base::sync_with_stdio(false); cin.tie(0);

    // Input
    int num;
    cin >> num;
    // Output
    string roman[9] = {"I", "II", "III", "IV", "V", "VI", "VII", "VIII", "IX"};
    if(num > 0 && num <= 9) { cout << roman[num - 1]; }
    else if(num < 0) { cout << "Error : Please input positive number"; }
    else { cout << "Error : Out of range"; }
    return 0;
}